astropy/coordinates/sites.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Observatories accessible by the `sites` module originate from the IRAF
+Observatory Database, and are stored in ``data/observatories.json``.
+Longitudes are listed with positive to the West.
```

Comment:
This is a comment that was written for `astroplan` which should probably be removed for `astropy`.